site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -952,6 +1047,23 @@ components:
             "ext":
                $ref: '#/components/schemas/Extensions'
 
+      SearchDevicesResult:
+         type: object
+         description: The result of a search request.
+         additionalProperties: false
+            properties:
+               "total":
+                  type: String
+                  description: The total number of results returned by the query, regardless of the "limit" set in the paging options.
+               "results":
+                  $ref: '#/components/schemas/Device'
```

Comment:
The search operation should return `#/components/schemas/DevicesList` (a list of devices) and not `#/components/schemas/Device`.